src/common/PriorityCache.h

```diff
@@ -68,6 +101,42 @@ namespace PriorityCache {
     // Get the name of this cache.
     virtual std::string get_cache_name() const = 0;
   };
+
+  struct Manager {
+    CephContext* cct = nullptr;
+    PerfCounters* logger;
```

Comment:
nit: std::unique_ptr<> would be better

Reply:
Going to leave this alone for now.  We use raw pointers for this everywhere else in the code so if we want to make a change let's rethink how logger works generally.